diff --git a/node_modules/nextra/dist/loader.js b/node_modules/nextra/dist/loader.js
index ed96dab..648170e 100644
--- a/node_modules/nextra/dist/loader.js
+++ b/node_modules/nextra/dist/loader.js
@@ -283,7 +283,7 @@ async function analyzeLocalizedEntries(currentResourcePath, defaultLocale) {
 
 async function loader (source) {
   const callback = this.async();
-  this.cacheable();
+  // this.cacheable();
   const options = loaderUtils.getOptions(this);
   const {
     theme,
diff --git a/node_modules/nextra/dist/nextra.js b/node_modules/nextra/dist/nextra.js
index 3a4c085..53ae743 100644
--- a/node_modules/nextra/dist/nextra.js
+++ b/node_modules/nextra/dist/nextra.js
@@ -40,6 +40,8 @@ async function buildStorkIndex(storkPath, locales) {
   for (const locale of locales) {
     const tomlFile = path.join(assetDir, `index-${locale}.toml`);
     let toml = await gracefulFs.promises.readFile(tomlFile, 'utf-8');
+    const index = toml.indexOf("[output]");
+    if(index < 0) return;
     toml += '[output]\n';
     toml += `filename = "${path.join(assetDir, `index-${locale}.st`)}"\n`;
     toml += `excerpts_per_result = 1\n`;
@@ -120,13 +122,15 @@ var index = ((...args) => (nextConfig = {}) => {
         }]
       });
       if (!config.plugins) config.plugins = [];
-      config.plugins.push({
-        apply: compiler => {
-          compiler.hooks.done.tap('buildStorkIndex', () => {
-            buildStorkIndex(STORK_PATH, locales);
-          });
-        }
-      });
+      if(nextraConfig.stork) {
+        config.plugins.push({
+          apply: compiler => {
+            compiler.hooks.done.tap('buildStorkIndex', () => {
+              buildStorkIndex(STORK_PATH, locales);
+            });
+          }
+        });
+      }
 
       if (typeof nextConfig.webpack === 'function') {
         return nextConfig.webpack(config, options);
